app/factories/flood_risk_engine/steps/form_object_factory.rb

```diff
@@ -38,11 +38,16 @@ def form_object_class_map
         private
 
         def setup_form_object(step)
-          "FloodRiskEngine::Steps::#{step.to_s.classify}Form".constantize
-        rescue
-          Rails.logger.debug(" No Form Object found for step [#{step}] - falling back to object_class map")
-          nil
+          form_name = "FloodRiskEngine::Steps::#{step.to_s.classify}Form"
```

Comment:
I think `form_name = "#{step.to_s.classify}Form".constantize` would work as we already in the `FloodRiskEngine::Steps` name space